### PR TITLE
Add cns usability test cases

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

**Comment by hongkailiu** [comment]
@mffiedler @vikaslaad 
Can you please take a look?

File: storage/cns_usability/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+```
+
+where `ec2-34-215-170-11.us-west-2.compute.amazonaws.com` is the master node of the OCP cluster.
+
+### Kill 2 glusterfs pods together
+
+In this case, we expect some loss of logging entires.
+
+```sh
+$ ansible-playbook -i "ec2-34-215-170-11.us-west-2.compute.amazonaws.com," storage/cns_usability/cns-usability-test-restart-2-glusterfs-pods.yaml
```
Comment (jeremyeder) [inline storage/cns_usability/README.md]: remove hostnames

Comment (jeremyeder) [inline storage/cns_usability/README.md]: can you throw a readme.md in to explain what these tests are and why we want them?

I also wonder if this sort of thing can be a tweak to the kraken framework that @wabouhamad and @chaitanyaenr merged last week?

**Comment by hongkailiu** [comment]
@jeremyeder 
I will address your comments in the next PR since this one has been merged already.

**Comment by hongkailiu** [comment]
Created PR: https://github.com/openshift/svt/pull/466

**Pull request merged**